src/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,432 @@
+import { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import { Stack } from '../Stack';
+import { Text } from '../Text';
+import { Placeholder } from '../_localComponents/Placeholder';
+import { Accordion } from './Accordion';
+
+const meta: Meta<typeof Accordion> = {
+  title: 'Components/Accordion',
+  component: Accordion,
+  tags: ['autodocs'],
+  parameters: {
+    layout: 'fullscreen',
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof Accordion>;
+
+const accordions = [
+  {
+    id: 0,
+    triggerCopy: 'Does Chromatic replace Jest or Enzyme?',
+    icon: 'cloudhollow',
+  },
+  {
+    id: 1,
+    triggerCopy: 'Can I get rid of my BrowserStack or Sauce subscription?',
+    icon: 'facehappy',
+  },
+  {
+    id: 2,
+    triggerCopy: 'How is this compared to Selenium, Cypress, or Playwright?',
+    icon: 'accessibilityalt',
+  },
+  { id: 3, triggerCopy: 'Why run visual tests in the cloud?', icon: 'heart' },
+];
+
+const PanelContent = ({ inverse }) => {
+  const color = inverse ? 'white' : 'slate800';
+
+  return (
+    <Text color={color}>
+      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quae, autem
+      expedita, in explicabo inventore quasi enim iusto voluptate quibusdam
+      eius, quas corporis. Aperiam vel minima nulla sint animi in ducimus.
+    </Text>
+  );
+};
+
+export const Base: Story = {
+  args: {
+    inverse: false,
+    iconName: 'arrowdown',
+    iconSize: 14,
+    triggerCopy: 'How is this compared to Selenium, Cypress, or Playwright?',
+  },
+  parameters: {
+    backgrounds: { default: 'light' },
+  },
+  render: (args) => {
+    return (
+      <Stack direction="column" marginY={10} marginX={10} gap={2}>
```

Comment:
It doesn't seem like `Stack` is an important part of the implementation. For components that live in other layouts that we also want to test in mobile viewports, the extra margins and gaps will mess with breakpoint definitions